tests/meta/demultiplex/test_demux_post_processing.py

```diff
@@ -739,24 +739,46 @@ def test_is_valid_sample_fastq_filename(demultiplex_context: CGConfig):
     assert demux_post_processing_api.is_valid_sample_fastq_filename(file_name)
 
 
-def test_get_sample_fastq_file_paths(demultiplex_context: CGConfig, tmpdir_factory):
+def test_get_valid_flowcell_sample_fastq_file_path(demultiplex_context, tmpdir_factory):
     # GIVEN a DemuxPostProcessing API
     demux_post_processing_api = DemuxPostProcessingAPI(demultiplex_context)
 
+    # GIVEN a flow cell directory
+    flow_cell_dir = Path(tmpdir_factory.mktemp("flow_cell"))
+
     # GIVEN some files in temporary directory
-    tmp_dir = Path(tmpdir_factory.mktemp("data"))
-    valid_file = tmp_dir / "file.fastq.gz"
-    invalid_file = tmp_dir / "Undetermined_file.fastq.gz"
-    valid_file.touch()
-    invalid_file.touch()
+    sample_dir = flow_cell_dir / "Unaligned" / "Project_sample" / "Sample_test"
```

Comment:
Will be great to test with a sample with the pattern "Sample_x_y"

Reply:
Added it :)